src/containers/Login/Login.js

```diff
@@ -19,26 +20,34 @@ export class Login extends Component {
   handleChange = (e) => {
     const { value, name } = e.target
     this.setState({
-      [name]: value,
+      [name]: value
     })
   }
 
   handleSubmit = async (e) => {
     e.preventDefault()
     const { email, password } = this.state
-    const users = await fetchData('http://localhost:3000/api/users')
-    let validUser = false
-    const currentUser = users.data.find(user => {
-      if (user.email === email && user.password === password) {
-        validUser = true
-        return user
-      } 
-    })
-    
-    if (validUser) {
-      this.props.loginUser(currentUser)
+    try {
+      const user = await loginUser({email, password})
```

Comment:
naming is confusing here.  loginUser vs. this.props.loginUser.  We should refactor those names